generators/client/templates/angular/src/main/webapp/scss/_global.scss

```diff
@@ -105,37 +29,12 @@ Logo styles
             width: 70px;
             display: inline-block;
             vertical-align: middle;
-            background: url("../images/logo-jhipster.png") no-repeat center center;
+            background: url("../content/images/logo-jhipster.png") no-repeat center center;
```

Comment:
can you see if its possible to malke it generate inside the content folder like other css?